pkgs/by-name/ob/obs-cmd/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "obs-cmd";
+  version = "0.17.4";
+
+  src = fetchFromGitHub {
+    owner = "grigio";
+    repo = "obs-cmd";
+    rev = "v${version}";
+    hash = "sha256-HCvIMIQZKzIkpYL9F9oM4xiE/gOeI+7dMj9QmhetHm4=";
+  };
+
+  cargoHash = "sha256-AQRjZH3WhZXU6NhDSCv4/HWz5un1nFtuzWPYSJA9XaE=";
+
+  meta = with lib; {
+    description = "A minimal CLI to control OBS Studio via obs-websocket.";
+    longDescription = ''
+      obs-cmd is an OBS Studio CLI for obs-websocket v5, the current obs-studio implementation.
+    '';
```

```suggestion
    description = "Minimal CLI to control OBS Studio via obs-websocket";
```
the long description is just a copy of the short one